apps/storybook-app/stories/atoms/ChatbotButton.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { Meta, StoryObj } from '@storybook/react';
+import ChatButton from '../../../nextjs-website/src/components/atoms/ChatButton/ChatButton';
+
+const meta: Meta<typeof ChatButton> = {
+  title: 'Atoms/ChatButton',
+  component: ChatButton,
+};
+
+export default meta;
+
+export const Showcase: StoryObj<typeof ChatButton> = {
+  args: {
+    chatOpen: false,
```

Comment:
`chatOpen` args looks like is never used, maybe did  you mean to use `isChatOpen` ? :thinking: